modules/virt-building-vm-containerdisk-image.adoc

```diff
@@ -6,25 +6,25 @@
 [id="virt-building-vm-containerdisk-image_{context}"]
 = Building a container disk image for {op-system-base} virtual machines
 
-You can build a custom {op-system-base-full} 8 OS image in `qcow2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map.
+You can build a custom {op-system-base-full} 9 OS image in `qcow2` format and use it to create a container disk image. You can store the container disk image in a registry that is accessible from your cluster and specify the image location in the `spec.param.vmContainerDiskImage` attribute of the DPDK checkup config map.
 
-To build a container disk image, you must create an image builder virtual machine (VM). The _image builder VM_ is a {op-system-base} 8 VM that can be used to build custom {op-system-base} images.
+To build a container disk image, you must create an image builder virtual machine (VM). The _image builder VM_ is a {op-system-base} 9 VM that can be used to build custom {op-system-base} images.
 
 .Prerequisites
-* The image builder VM must run {op-system-base} 8.7 and must have a minimum of 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory.
-* You have installed the image builder tool and its CLI (`composer-cli`) on the VM.
+* The image builder VM must run {op-system-base} 9.4 and must have a minimum of 2 CPU cores, 4 GiB RAM, and 20 GB of free space in the `/var` directory.
+* You have [installed the image builder](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_a_customized_rhel_system_image/installing-composer_composing-a-customized-rhel-system-image#composer-system-requirements_installing-composer) tool and its CLI (`composer-cli`) on the VM.
```

Comment:
The syntax of this cross reference is not correct. Regardless, we cannot add an internal cross reference in a "module" file because of tooling limitations. That is why all of our links are in the Additional resources section. Maybe you can change it to something like:
```suggestion
* You have installed the image builder tool and its CLI (`composer-cli`) on the VM. For more information, see "Additional resources".
```